client/src/components/Project.js

```diff
@@ -42,11 +45,21 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const Project = ({ data, gridSize }) => {
-   const format = d3Format(',');
+  const format = d3Format(',');
   const classes = useStyles();
+  const history = useHistory();
+
   const { fundingGoal, images, industry, title } = data;
   return (
-    <Grid item xs={gridSize}>
+    <Grid
+      item
+      xs={gridSize}
+      onClick={() => {
+        history.push({
+          pathname: `/project/${data._id}`,
+        });
+      }}
+    >

```

Comment:
I was thinking of completely taking out this Grid component and wrapping the Project component with it wherever we end up using it. In the profile page we need it to route the user to the 'edit project' page if they click on it. However because we use this Project component in the Explore page too we need it to route to the individual project page as well.

Rather than trying to make it too complex with different params and what not I figured we could just wrap the <Grid /> component around <Project /> with the correct paths and links we need for wherever we end up using it. For example on the profile page I'd have something like this:
```// Profile Page
{projects.map(project => (
  <Grid
    item
    xs={6}
    onClick={handleProjectClick}
  >
    <Project data={project} />
  </Grid>
  )
}
```
What do you think?

Reply:
That's actually a good point, so depending on where the user clicks the project, it could to be redirected to 2 different pages. So, instead of dealing with onClick inside the Project component, you're saying we can have individual onClicks in the Profile and Explore pages right?

Reply:
Exactly! And we'll put the onClicks inside the Grid component that we wrap the project component with when we map over the projects.